### PR TITLE
Add .htaccess to automatically forward HTTP to HTTP

### PR DESCRIPTION
**Pull request by davorbonaci**
R: @dhalperi @jbonofre

### CODE REVIEW

**Comment by dhalperi** [comment]
I feel like we want something like # 4 here: https://moz.com/blog/htaccess-file-snippets-for-seos

But I need to read up on `htaccess` more.

**Comment by dhalperi** [comment]
From there:

```
# Ensure www on all URLs.
RewriteCond %{HTTP_HOST} ^example.com [NC]
RewriteRule ^(.*)$ http://www.example.com/$1 [L,R=301]
```

So maybe:

```
# Ensure www on all URLs.
RewriteCond %{HTTP_HOST} beam.incubator.apache.org [NC]
RewriteRule ^(.*)$ http://beam.apache.org/$1 [L,R=301]
```

**Comment by dhalperi** [comment]
And may need to set `RewriteEngine On`

**Comment by dhalperi** [comment]
Another comment to test Jenkins.

File: src/.htaccess [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+# This is a 301 (permanent) redirect from HTTP to HTTPS.
+# Goal: use HTTPS instead of HTTP.
+
+# The next rule applies contionally:
+# * the host is "beam.apache.org",
+# * the host comparison is case insensitive (NC),
+# * the server port is 80, i.e., HTTP.
+RewriteCond %{HTTP_HOST} ^beam\.apache\.org [NC]
+RewriteCond %{SERVER_PORT} 80
```
Comment (dhalperi) [inline src/.htaccess]: RewriteCond %{HTTPS} !on

File: src/.htaccess [hunk 2]
```diff
@@ -0,0 +1,15 @@
+RewriteEngine On
+
+# This is a 301 (permanent) redirect from HTTP to HTTPS.
+
+# The next rule applies contionally:
```
Comment (dhalperi) [inline src/.htaccess]: conditionally.

**Comment by jbonofre** [comment]
Sorry for the delay: LGTM

**Pull request merged**